src/coreclr/src/debug/createdump/main.cpp

```diff
@@ -40,6 +40,15 @@ int __cdecl main(const int argc, const char* argv[])
     int exitCode = 0;
     int pid = 0;
 
+#ifdef __APPLE__
+    char* enabled = getenv("COMPlus_DbgEnableElfDumpOnMacOS");
```

Comment:
We want new environment variables to use a DOTNET_ prefix not a COMPlus_ prefix, because the latter prefix doesn't mean anything to customers. Would DOTNET_DbgEnableElfDumpOnMacOS work?

Reply:
I know there's several COMPlus_DbgXXX already, so could go either way -- if we use the old format again, we maybe will want to support both in future.

Reply:
The rest of the createdump related env vars are COMPlus_ so I think this one should be too and it is really for internal use.